lib/imagec/imagec.go

```diff
@@ -69,14 +72,15 @@ type ImageC struct {
 }
 
 // NewImageC returns a new instance of ImageC
-func NewImageC(options Options, strfmtr *streamformatter.StreamFormatter, getArchiveReader GetArchiveReader) *ImageC {
+func NewImageC(options Options, strfmtr *streamformatter.StreamFormatter, proxy proxy.VicArchiveProxy) *ImageC {
 	return &ImageC{
 		Options:        options,
 		sf:             strfmtr,
 		progressOutput: strfmtr.NewProgressOutput(options.Outstream, false),
 		Pusher: Pusher{
-			streamMap:     make(map[string]*ArchiveStream),
-			ArchiveReader: getArchiveReader,
+			Simulate:     true,
```

Comment:
Is this variable used for the sanity check of layer tars when running imagec in standalone mode? Should it be false by default?

Will you have another PR to add this variable as a flag option in `cmd/imagec`?